clientBuilder.sslSocketFactory(SSLSocketFactory) not supported on JDK 9+

We are seeing this exception when trying to run our service on Java 11 (LTS):

```
Caused by: java.lang.UnsupportedOperationException: clientBuilder.sslSocketFactory(SSLSocketFactory) not supported on JDK 9+                      
        at okhttp3.internal.platform.Jdk9Platform.trustManager(Jdk9Platform.java:81)                                                               
        at okhttp3.internal.platform.Platform.buildCertificateChainCleaner(Platform.java:176)                                                      
        at okhttp3.OkHttpClient$Builder.sslSocketFactory(OkHttpClient.java:655)                                                                    
        at io.featureflow.client.core.EventSource.<init>(EventSource.java:65)                                                                      
        at io.featureflow.client.core.FeatureControlStreamClient.start(FeatureControlStreamClient.java:62)                                         
        at io.featureflow.client.FeatureflowClient.<init>(FeatureflowClient.java:78)                                                               
        at io.featureflow.client.FeatureflowClient$Builder.build(FeatureflowClient.java:221)                                                       
...
```

--- a/pom.xml
+++ b/pom.xml
@@ -16,7 +16,7 @@
         <httpclient.version>4.5.2</httpclient.version>
         <joda-time.version>2.9.3</joda-time.version>
         <gson.version>2.3.1</gson.version>
-        <okhttp.version>3.2.0</okhttp.version>
+        <okhttp.version>3.14.6</okhttp.version>
         <pubcontrol.version>1.0.7</pubcontrol.version>
         <fanout.version>1.0.0</fanout.version>
         <maven-source-plugin.version>3.0.1</maven-source-plugin.version>

--- a/src/main/java/io/featureflow/client/core/EventSource.java
+++ b/src/main/java/io/featureflow/client/core/EventSource.java
@@ -62,7 +62,7 @@ public class EventSource implements ConnectionHandler, Closeable {
             throw new RuntimeException(e);
         }
         this.client = new OkHttpClient().newBuilder()
-                .sslSocketFactory(sslSocketFactory)
+                .sslSocketFactory(sslSocketFactory, trustManager)
                 .readTimeout(0L, TimeUnit.SECONDS).writeTimeout(0L, TimeUnit.SECONDS).connectTimeout(0L, TimeUnit.SECONDS).retryOnConnectionFailure(true).build();
     }
 

